src/use-image.hook.js

```diff
@@ -9,8 +9,10 @@ export function useImage({processImgElement = noop, fileBlobToUrl = defaultFileB
   const {performCommandWithValue} = useDocumentExecCommand('insertImage')
   const richTextContext = useContext(RichTextContext)
   const fileInputRef = useRef(null)
+  const dataUrls = React.useRef({})
```

Comment:
the serialize function is synchronous, but getting the inline image as a big long data url is not. So we do it when the file is selected, not when serialize is called.